docs/references/pysdk_api_reference.md

```diff
@@ -1454,34 +1350,77 @@ table_obj.update("c1 > 2", [{"c2": 100, "c3": 1000}])
 ```python
 table_obj.output(columns)
 ```
-Specify the columns to display in the search output, or perform aggregation operations or arithmetic calculations. 
+Specifies the columns to display in the search results or performs aggregation or arithmetic operations on them. This method allows you to customize the output of your query by selecting specific columns, applying aggregation functions, or performing arithmetic operations on the specified columns.
 
-### Parameters
+:::tip NOTE
+This method specifies the projection columns for the current table but does not directly produce displayable data. To display the query results, use `output()` in conjunction with methods like `to_result()`, `to_df()`, `to_pl()`, `to_arrow()`, or others that materialize the data.
+:::
 
-#### columns: `list[str]`, *Required*
+### Parameters
 
-Must not be empty. Supported aggragation functions:
+#### columns: `list[str]` (non-empty), *Required*
+
+A non-empty list of strings specifying the columns to include in the output. Each string in the list can represent:
+
+- A user-defined column name: Directly specify the names of the columns to include in the output, e.g., `"body"`.
+- All user-defined columns: Use a wildcard `"*"` to select all columns.
+- A special system column: system-generated columns include:
+  - `_row_id`
+  - `_score`
+  - `_similarity`
```

Comment:
similarity, used by IP and cosine metric of dense vector. What about sparse and tensor?